spring-data-jdbc/src/test/resources/testcontainers.properties

```diff
@@ -0,0 +1 @@
+oracle.container.image=wnameless/oracle-xe-11g@sha256:825ba799432809fc7200bb1d7ef954973a8991d7702a860c87177fe05301f7da
```

Comment:
We must not use this docker image since it is not on supported by Oracle and quite possibly illegal to use. 

Please only use official Oracle images: https://github.com/oracle/docker-images

Reply:
That might make things a lot easier (e.g. ID columns). Somehow previously I only found images that one would have to build locally. I assume requiring a docker login for pulling the Oracle images before running the Oracle tests is ok?

Reply:
Yes, a login to Docker is ok.